src/WebJobs.Script.WebHost/Diagnostics/EventGenerator.cs

```diff
@@ -75,7 +251,7 @@ public void LogFunctionDetailsEvent(string siteName, string functionName, string
             {
                 if (IsEnabled())
                 {
-                    WriteEvent(57908, siteName, functionName, inputBindings, outputBindings, scriptType, isDisabled);
+                    WriteEvent(59708, siteName, functionName, inputBindings, outputBindings, scriptType, isDisabled);
```

Comment:
Why are we changing this event ID? I do see that all the others are 59xxx, but this may cause issues.